.github/workflows/.schema-spy.yml

```diff
@@ -0,0 +1,53 @@
+name: .Schema Spy
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        type: string
+        required: false
+        default: dev
+jobs:
+  generate-schema-spy:
+    name: Generate SchemaSpy Documentation
+    runs-on: ubuntu-24.04
+    services:
+      postgres:
+        image: postgis/postgis:16-3.4
+        env:
+          POSTGRES_DB: site
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: default
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+      - name: Migrations
+        uses: docker://ghcr.io/bcgov/nr-site-registry/backend/sites:${{ inputs.tag }}
```

Comment:
I believe the migration image is called `sites-migration:dev`, at least in my original setup. However I think we could use the shared image with a different CMD

Reply:
good catch, need to change it, I will do it along with the other change I am doing around frontend config, I just created the reusable action but currently it is not called from anywhere.